packages/opentelemetry-sdk-trace-web/src/utils.ts

```diff
@@ -85,7 +85,12 @@ export function addSpanNetworkEvents(
   addSpanNetworkEvent(span, PTN.DOMAIN_LOOKUP_START, resource);
   addSpanNetworkEvent(span, PTN.DOMAIN_LOOKUP_END, resource);
   addSpanNetworkEvent(span, PTN.CONNECT_START, resource);
-  addSpanNetworkEvent(span, PTN.SECURE_CONNECTION_START, resource);
+  if (
+    hasKey(resource as PerformanceResourceTiming, 'name') &&
+    (resource as PerformanceResourceTiming)['name'].startsWith('https:')
```

Comment:
Technically, it is also possible to have `http://` with port `:443`, but I'm happy with saying that we generally only support direct `https:` as this "should" catch most people.